TestMod/TestMod.cs

```diff
@@ -272,7 +285,7 @@ private void AddSkills()
             });
 
             // Test adding skills from JSON
-            SkillManager.Instance.AddSkillsFromJson("TestMod/Assets/skills.json");
+            //SkillManager.Instance.AddSkillsFromJson("TestMod/Assets/skills.json");
```

Comment:
Shouldn't we leave this in to test the JSON skills as well?

Reply:
my bad, nicely spotted